pallets/parachain-staking/src/weights.rs

```diff
@@ -121,50 +125,50 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking InflationConfig (r:1 w:1)
 	#[rustfmt::skip]
 	fn set_staking_expectations() -> Weight {
-		Weight::from_ref_time(27_136_000 as u64)
+		Weight::from_ref_time(27_580_000 as u64)
```

Comment:
Many of these updates are irrelevant and might impact precompiles. I suggest only including the new ones.

Also, were these run on official hardware? I can run them if needed, just let me know.

Reply:
Yup these have been run on the `benchmarking` node, I cherry picked the weights now that should've changed.